packages/terra-demographics-banner/src/DemographicsBanner.scss

```diff
@@ -0,0 +1,79 @@
+@import '~terra-mixins';
+@import './variables';
+@import './mixins';
+
+.terra-DemographicsBanner {
+  background: $demographics-background;
+  color: $demographics-foreground;
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 0.714em;
+  padding: 0.714em;
+
+  @media screen and (max-width: $terra-small-breakpoint) {
+    flex-direction: column;
+  }
+}
+
+.terra-DemographicsBanner--person-name {
+  margin: 0;
+}
+
+.terra-DemographicsBanner--person-details {
+  flex: 1;
+  min-width: 0;
+  white-space: nowrap;
```

Comment:
Nit: move whitespace below text-overflow. This is causing style lint and I think causing CI to fail.